### PR TITLE
Prod IAC related release changes

### PR DESCRIPTION
**Pull request by mikeAdamss**
## What this is

Necessary changes for the cms to run on the new IAC environments.

- Add a startup script (that runs before zope starts) to force zope to recognise environment variables (there's probably a better way long term, but all variations of everything I could find failed - this works).
- remove the cloudbuild definitions (now handled elsewhere via terraform)
- update docker compose and makefile for the new world, switch to docker-compose healchecks via wget as we're having issues with the curl based ones.
- bumped authomatic to a newer version
- minor tidy ups

trialled extensively in sandbox and staging already.

## How to review

sanity check

### CODE REVIEW

**Comment by eldeal** [comment]
All looks sensible to me as a direction! 👍

**Review by eldeal — APPROVED** [review]
Other than minor doc things, this looks good to me

File: plone-5/scripts/README.md [hunk 1]
```diff
@@ -0,0 +1,19 @@
+# Scripts
+
+Scripts for use on a _running_ plone server.
+
+## Prestart
+
+Scripts that need to be ran _before_ the server starts should be called via ./prestart.sh
```
[the remark this answers is not in the document]
Comment (mikeAdamss) [reply plone-5/scripts/README.md:7]: sharp eyes 😄 , I actually deleted this readme locally for much the same reason (its muddled) but forgot to push that.

I did have some rigging in here for running monkey patches and the like agains the plone server, but worried it was gonna encourage hackery over fixing root problems. Will just remove the readme, the comments in prestart.sh cover whats going on here pretty cleanly.

[the remark this answers is not in the document]
Comment (mikeAdamss) [reply plone-5/scripts/README.md:7]: the direct calling of the python api with scripts wont work anyway without a monkey patch (long story and I need to add a card, but its why this got confused)

**Pull request merged**